core/services/wifi/WifiManager.py

```diff
@@ -183,6 +183,19 @@ async def connect_to_network(self, network_id: int, timeout: float = 10.0) -> No
             await self.wpa.send_command_save_config()
             await self.wpa.send_command_reconfigure()
             await self.wpa.send_command_reconnect()
+            t0 = time.time()
+            while True:
+                status = await self.status()
+                is_connected = "wpa_state" in status and status["wpa_state"] == "COMPLETED"
+                if is_connected:
+                    current_network = await self.get_current_network()
+                    if current_network and current_network.networkid == network_id:
```

Comment:
this missing underscore in `networkid` :eyes:, it appears to be a style problem.
we can fix that in another PR, just reminding that we need to update the frontend as well.

Reply:
It comes like that from wpa_supplicant in fact. We just parse it to a `SavedWifiNetwork`, without changes. Check line 130.